frontend/src/app/dora/service/api.service.ts

```diff
@@ -46,6 +56,26 @@ export class ApiService {
     });
   }
 
+  verifyLinearBoard({
+    type,
+    teamName,
+    teamId,
+    startTime,
+    endTime,
+    token,
+  }: {
+    type: string;
+    teamName: string;
+    teamId: string;
+    startTime: string;
+    endTime: string;
+    token: string;
+  }) {
+    return this.httpClient.get(`${this.baseUrl}/kanban/verify`, {
+      params: { token, type: type.toLowerCase(), teamName, teamId, startTime, endTime },
+    });
```

Comment:
This is related to the comment above. Abstracting a param type is also beneficial to this
```
verifyLinearBoard(params: LinearBoardParam) {
    return this.httpClient.get(`${this.baseUrl}/kanban/verify`, {
      params: { ...params, type: type.toLowerCase() },
    });
}
```

Reply:
same as above.